test/pkg_finish_test.rb

```diff
@@ -341,6 +341,14 @@
 
       context "if libzypp's minimalistic configuration is enabled" do
         let(:minimalistic_libzypp_config) { true }
+        let(:destdir) { tmpdir }
+
+        before do
+          conf_file = Yast::Packager::CFA::ZyppConf::PATH[1..-1]
+          conf_path = conf_file.split("/")[0..-2].join("/")
```

Comment:
This looks too complicated and it just returns the "etc/zypp" path. What about simplifying it to:

```suggestion
          conf_path = File.dirname(Yast::Packager::CFA::ZyppConf::PATH)
```

`File.join` does not mind if there is an extra `/`...